final-project/manUserInterviews.tex

```diff
@@ -69,13 +69,134 @@ \section{Interview: Jay Creamer}
+
+Yes, less often than sharing them though.
+
+\textbf{Tell me about the last time you had a watch party.}
+
+Last online party, me and my friends were watching Nintendo Direct on Youtube together. We were all watching separately though and we were messaging each other about it. We didn't have a way to watch all together at the time, Discord was being weird.
+
+\textbf{How do you feel about OTT as a platform?}
+
+I think it's really cool, I like that it's easy to input videos and it's useful to have playlists. I don't know many other video sharing websites that have that feature. I like that you can sign in with your Discord account. Like that it supports many video platforms like Youtube, Vimeo. It's a good utility tool.
+
+\textbf{How often do you encounter problems with OTT? Specifically, down time, losing room state, or unresponsiveness.}
+
+I haven't used it that much but I didn't face any problems.
+
+\textbf{Do you use permanent rooms?}
+
+Yes, seems neat.
+
+\textbf{Would you say that using permanent rooms are more convenient than temporary rooms?}
+
+Yes. If you're doing it with a consistent group, everyone is aware of the same link so you don't have to keep sending links. And rules are more consistent so you don't have to adjust the settings every time.
+
+\textbf{If you wanted to watch a video with a friend, but OTT was not available, what would you do?}
+
+I would probably use Discord.
+
+\textbf{If OTT kept having problems, would you continue to use it?}
+
+If there were consistent problems, probably not. There are a lot of video sharing platforms out there. If the problems were fixed, I'd probably go back.
+
+
+\section{Interview: Jakob Gibson}
+
+User: Jakob Gibson (Friend, Student at Stevens Institute of Technology) \\
+Interviewer:
+
+\textbf{Do you often share videos with your friends?}
+
+Yes. Depends, like once a day.
+
+\textbf{Have you ever watched videos with your friends?}
+
+Yes.
+
+\textbf{Tell me about the last time you had a watch party.}
+
+If we're talking in-person, those happen all the time. If we're talking online, probably on Discord, somebody was streaming what they were watching. It's usually impromptu.
+
+\textbf{How do you feel about OTT as a platform?}
+
+I think it's an interesting idea. Having the ability to have synchronized rooms is cool, there are extensions exists for that but they can be kinda janky.
+
+\textbf{How often do you encounter problems with OTT? Specifically, down time, losing room state, or unresponsiveness.}
+
+I haven't used it enough. N/A
+
+\textbf{Do you use permanent rooms?}
+
+Yes, if there was the same group of people I'd share it with or just need to share a video with a couple of people.
+
+\textbf{Would you say that using permanent rooms are more convenient than temporary rooms?}
+
+Probably less convenient because of sign in, but it means you don't have to share links. They're around equally convenient, there's benefits and drawbacks to both.
```

Comment:
Interesting, because this is not actually true. You can create a permanent room without being logged in to OTT.

Reply:
ah they probably didn't see the option lol